### ISSUE TITLE
module JaSerializer.Params is not loaded and could not be found

### ISSUE THREAD

**Issue by nicooga** [AgilionApps/ja_serializer#102]
This is what I get when trying to call or `require JaSerializer.Params`.

This are my deps:

~~~elixir
    [{:phoenix, "~> 1.1.4"},
     {:postgrex, ">= 0.0.0"},
     {:phoenix_ecto, "~> 2.0"},
     {:gettext, "~> 0.9"},
     {:cowboy, "~> 1.0"},
     {:ja_serializer, "~> 0.8.1"},
     #{:monetized, git: "git@github.com:nicooga/monetized.git"},
     #{:cors_plug, "~> 1.1"}
   ]
~~~

I removed all non phoenix generated deps. `mix` always compiles without visible error, but I can't access the module.

**Comment by alanpeabody** [comment]
Hey @nicooga, JaSerializer.Params is in master and will be released in 0.9.0 soon!

**Comment by nicooga** [comment]
I see, thanks for the answer. I switched to using master from github and works.
The module Params was present on the source using "~> 0.8.1" though.
I don't think I understood how you tell mix/hex to compile and make some modules available from your package, a matter for research.
